### PR TITLE
Add fasm.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: packages/fasm/KagamiBuild [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Description: An assembler for x86 processors.
+# URL:         https://flatassembler.net
+# Maintainer:  ixavoz, ixavoz at disroot dot com
+# Section:     devel
+name=fasm
```
Comment (protonesso) [inline packages/fasm/KagamiBuild]: leave space here

File: packages/fasm/KagamiBuild [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+version=1.73.22
+release=1
+source=("https://flatassembler.net/fasm-$version.tgz")
+build()
```
Comment (protonesso) [inline packages/fasm/KagamiBuild]:
```
build() {
```
not
 ```
build()
{
```

File: packages/fasm/KagamiBuild [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+{
+    # I don't know how to write shell scripts.
+    "$SRC"/fasm/fasm.x64 "$SRC"/fasm/source/Linux/fasm.asm
```
Comment (protonesso) [inline packages/fasm/KagamiBuild]: replace spaces with tabs

**Comment by protonesso** [comment]
anyway the package itself looks good

**Comment by protonesso** [comment]
but if this package is x86_64 only add

```
if [ "$BARCH" != "x86_64" ]; then
	die "fasm is not supported on $BARCH"
fi
```

File: packages/fasm/KagamiBuild (hunk 1 above)
Comment (ghost) [reply packages/fasm/KagamiBuild:6]: Okay.

File: packages/fasm/KagamiBuild (hunk 2 above)
Comment (ghost) [reply packages/fasm/KagamiBuild]: It's my style. Fix for consistency to the rest of KagamiBuild files anyway.

File: packages/fasm/KagamiBuild (hunk 3 above)
Comment (ghost) [reply packages/fasm/KagamiBuild]: I prefer spaces to tabs. Fix for consistency to the rest of KagamiBuild files anyway.

File: packages/fasm/KagamiBuild [hunk 4]
```diff
@@ -2,14 +2,20 @@
...
 version=1.73.22
 release=1
 source=("https://flatassembler.net/fasm-$version.tgz")
-build()
-{
-    # I don't know how to write shell scripts.
-    "$SRC"/fasm/fasm.x64 "$SRC"/fasm/source/Linux/fasm.asm
-    mkdir -p "$PKG"/usr/bin/
-    mv "$SRC"/fasm/source/Linux/fasm "$PKG"/usr/bin/
+
```
Comment (protonesso) [inline packages/fasm/KagamiBuild]: :+1:

**Pull request merged**